homeassistant/components/samsungtv/media_player.py

```diff
@@ -225,7 +226,7 @@ def send_key(self, key):
             self._remote = None
             LOGGER.debug("Failed sending command %s", key, exc_info=True)
             return
-        except OSError:
+        except OSError, WebSocketException:
```

Comment:
LGTM!

Reply:
(might want to start some comments explaining example scenarios where/why these exceptions can occur)